news/_posts/2021-06-21-Poudel-defense.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: news
+title: Ranjita Poudel defends dissertation
+author: Taylor Salo
+category: news
+tags: []
+image: /assets/images/team/ranjita-poudel.jpg
+excerpt: Graduate student Ranjita Poudel has successfully defended her dissertation
+---
+
+[Ranjita Poudel](/team/poudel-ranjita) successfully defended her dissertation,
+"Neurocognitive Mechanisms Associated with Real-world Financial Savings Among Individuals from Lower Income Households", on June 21, 2021,
+and has now received her Ph.D. in Cognitive Neuroscience. She will officially graduate in the 2021-2022 Fall semester.
+
+Congratulations Ranjita!
```

Comment:
@rpoudel1 is there anything you'd like to see changed here to make it more customized? Otherwise it looks really similar to Katie and Jessica's announcements.